src/main/java/haveric/recipeManager/tools/Version.java

```diff
@@ -52,6 +55,19 @@ private static boolean supports1_11() {
 
         return supports;
     }
+    private static boolean supports1_10() {

```

Comment:
:D Sounds good. I'll get a revised push shortly to address these discussed points.